openpathsampling/analysis/tis_analysis.py

```diff
@@ -65,27 +60,22 @@ def all_movers(self):
             all_movers += self.movers[movetype]
         return all_movers
 
-    def _assign_sample(self, sample):
-        if sample.ensemble in self.all_ensembles:
-            try:
-                self._samples_by_ensemble[sample.ensemble].append(sample)
-            except KeyError:
-                self._samples_by_ensemble[sample.ensemble] = [sample]
-            try: 
-                self._samples_by_id[sample.replica].append(sample)
-            except KeyError:
-                self._samples_by_id[sample.replica] = [sample]
+    # TODO: @dwhs can this function be removed?
+    # def _assign_sample(self, sample):
+    #     if sample.ensemble in self.all_ensembles:
+    #         try:
+    #             self._samples_by_ensemble[sample.ensemble].append(sample)
+    #         except KeyError:
+    #             self._samples_by_ensemble[sample.ensemble] = [sample]
+    #         try:
+    #             self._samples_by_id[sample.replica].append(sample)
+    #         except KeyError:
+    #             self._samples_by_id[sample.replica] = [sample]
 
-    @property
-    def all_movers(self):
-        all_movers = []
-        for movetype in self.movers.keys():
-            all_movers += self.movers[movetype]
-        return all_movers
-
-    @property
-    def all_ensembles(self):
-        return self.ensembles
+    # TODO: can this be removed? Only used in _assign_sample
+    # @property
+    # def all_ensembles(self):
+    #     return self.ensembles
```

Comment:
No. This is overridden by other transitions, but that gives the default behavior.